students/13th/gyungminbaek/js/login.js

```diff
@@ -0,0 +1,18 @@
+function activateLogin() {
+    console.log("activate")
+    const loginButton = document.querySelector('.button');
+    
+    const ID = document.querySelector('.inputID').value;
+    const PW = document.querySelector('.inputPW').value;
+
+    (ID.length>=1)&&(PW.length>=6) ?
+    loginButton.style.backgroundColor='#0095F6':
+    loginButton.style.backgroundColor='#B2dffc';
+  }
+
+document.querySelector('.inputID').addEventListener('keyup', function(){
+    activateLogin();
+})
+document.querySelector('.inputPW').addEventListener('keyup', function(){
+    activateLogin();
+})
```

Comment:
그리고 질문사항에서 동일 이벤트인데 ID와 PW target을 같이 주고 싶은경우 `ID.PW.addEventListener`가 안되는 이유는 이벤트 타겟인 ID와 PW가 각각의 객체타입으로 addEventListener는 메서드를 사용하기 때문입니다. 그래서 문법적으로 `ID.PW`는 맞지 않습니다. 
동일 이벤트를 가지기때문에 코드를 통합해서 사용하길 원한다면 가장 최상위 부모인 ('.form')에서 이벤트 캡쳐링을 해주면 적용가능합니다.  다른 방법으로는 switch문을 사용해볼수도 있겠죠? 
더 공부해보시려면 event target, event bubbling, event capture 를 검색해보세요~

```suggestion
const form = document.querySelector(".form");

form.addEventListener(
  "keyup",
  function () {
    activateLogin();
  },
  true
);
```